### PR TITLE
feat: add snippets for using service account access token

### PR DESCRIPTION
**Pull request by kroikie**
Add snippet for using service account access token

- Move snippet for generating service account access token to Auth snippets.
- Add snippet for using service account access token in an FCM request without Admin SDK.

### CODE REVIEW

File: admin/src/main/java/com/google/firebase/example/Main.java [hunk 1]
```diff
@@ -22,22 +22,6 @@
 import java.io.IOException;
 
 public class Main {
```
Comment (chong-shao) [inline admin/src/main/java/com/google/firebase/example/Main.java:24]: Maybe this entire file can be removed (can do it separately in a different PR)?

**Comment by chong-shao** [comment]
LGTM. Left a minor comment.

**Review by egilmorez — APPROVED** [review]
